### PR TITLE
Feature sensor data to greyscale image

### PR DESCRIPTION
**Pull request by jarivandam**
Tests are not working because hwilb::cout is not supported in the tests.

### CODE REVIEW

File: src/color_pixel.hpp [hunk 1]
```diff
@@ -50,5 +50,14 @@ class ColorPixel {
      */
     int getGreen() const;
 };
-
+/**
+ * @brief Operator== for two color pixels
+ * [BLANK]
```
Comment (BIGStrawberry) [inline src/color_pixel.hpp]: I don't think you're supposed to actually leave "[blank]" in the doxygen comments?

Comment (BIGStrawberry) [reply src/color_pixel.hpp]: (I've seen this at a few places)

File: src/ycbcr.cpp [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#include "ycbcr.hpp"
```
Comment (BIGStrawberry) [inline src/ycbcr.cpp:1]: Will the class YCbCr be getting different functions than just getters and setters for its values? If not; I think you're better off putting those values in a struct.

**Review by BIGStrawberry — COMMENTED** [review]
Will the class YCbCr be getting different functions than just getters and setters for its values? If not; I think you're better off putting those values in a struct.

I don't think you're supposed to actually leave "[blank]" in the doxygen comments?

File: src/ycbcr.cpp (hunk 2 above)
Comment (jarivandam) [reply src/ycbcr.cpp:1]: I think there is a large chance some operators will be added in the future. Also the other pixel types are classes so I think we should keep it a class for consistency.

**Pull request merged**